pkgs/by-name/ko/koji/package.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, perl
+, udev
+, openssl
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "koji";
+  version = "2.2.0";
+
+  src = fetchFromGitHub {
+    owner = "its-danny";
+    repo = "koji";
+    rev = version;
+    hash = "sha256-2kBjHX7izo4loJ8oyPjE9FtCvUODC3Sm4T8ETIdeGZM=";
+  };
+
+  cargoHash = "sha256-owppYDt0YdWoDvfmzVfiIPjLgTAT9eTI1LpRr4Y3XQA=";
+
+  nativeBuildInputs = [
+    pkg-config
+    perl
+    udev
+  ];
+
+  buildInputs = [
+    openssl
```

For smaller closure size:
```suggestion
    opemssl.dev 
```